client/state/selectors/get-selected-site-id.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns selected site id or null if no site is selected
+ *
+ * @param  {Object}   state  Global state tree
+ * @return {?Boolean}        Whether site is a Domain-only site
```

Comment:
It is valid [as far as I can tell
](http://usejsdoc.org/tags-type.html)

![screen shot 2017-01-31 at 13 48 46](https://cloud.githubusercontent.com/assets/326402/22463772/3774d420-e7bc-11e6-837d-e74d24ead44a.png)